packages/data-fetcher/.env.example

```diff
@@ -12,4 +12,8 @@ RPC_CALLS_RETRIES_MAX_TOTAL_TIMEOUT=120000
 RPC_CALLS_CONNECTION_TIMEOUT=60000
 RPC_CALLS_CONNECTION_QUICK_TIMEOUT=10000
 
+RPC_BATCH_MAX_COUNT=1
```

Comment:
Same question here? Are these good defaults? As most people will just make this their defaults.